Cakes-Co/Lato/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2010-2014 by tyPoland Lukasz Dziedzic (team@latofonts.com) with Reserved Font Name "Lato"

```

Comment:
You absolutely can use custom fonts, but the number of font files in this PR makes it a little difficult to review...

Reply:
Thank you. I will keep that in mind next time.